src/helix/impl/props.cljc

```diff
@@ -72,6 +72,47 @@
 (defn merge-obj [o1 o2]
   #?(:cljs (js/Object.assign o1 o2)))
 
+(defn seq-to-class [class]
+  (if (sequential? class)
+    (->> class
+         (map str)
+         (string/join " "))
```

Comment:
I think that we should filter out `nil` values here too.

Reply:
done